api/client/stats_helpers.go

```diff
@@ -150,7 +154,7 @@ func (s *containerStats) Collect(cli client.APIClient, streamStats bool, waitFir
 				s.mu.Lock()
 				s.err = err
 				s.mu.Unlock()
-				return
+				continue
```

Comment:
Up above we'll show zeros if we don't get a response within 2 seconds, I'm wondering if we should show --'s now instead?